### PR TITLE
feat: add alchemy as rpc url for opt sepolia

### PR DESCRIPTION
**Pull request by avasisht23**
## PR-Codex overview
This PR focuses on adding RPC URLs to support Optimism Sepolia chain.

### Detailed summary
- Added RPC URLs for Optimism Sepolia chain
- Added RPC URLs for Alchemy provider

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by tmm** [comment]
FYI [Viem v2](https://github.com/wevm/viem/pull/1429) will be released soon and will no longer accept these type of pull requests because we are moving away from hardcoding provider URLs and are only going to keep track of public resources.

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a changeset with `bun changeset`

**Comment by avasisht23** [comment]
added changeset @tmm

File: .changeset/new-drinks-admire.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"viem": minor
```
Comment (tmm) [inline .changeset/new-drinks-admire.md:2]:
```suggestion
"viem": patch
```

File: .changeset/new-drinks-admire.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"viem": patch
+---
+
+feat: add alchemy as rpc url for opt sepolia
```
Comment (tmm) [inline .changeset/new-drinks-admire.md:5]:
```suggestion
Added RPC URLs to Optimism Sepolia
```

**Pull request merged**